src/connectors/compliance/mock.js

```diff
@@ -178,7 +178,9 @@ module.exports = new class extends Connector {
         super(module);
     }
 
-    getRule (id) {
+    // Mock data doesn't contain platform or ssgVersion and there isn't value in adding it since we're just testing a mock of Compliance service
```

Comment:
I'd love to review all of our test data and coverage in general, but that's a huge undertaking (and beyond the scope of this PR)...